File: sdks/php/src/Taxes/Requests/AssignToPlanTaxesRequest.php
<?php

namespace NovaBilling\Taxes\Requests;

use NovaBilling\Core\Json\JsonSerializableType;
use NovaBilling\Types\AssignTaxDto;

class AssignToPlanTaxesRequest extends JsonSerializableType
{
    /**
     * @var AssignTaxDto $body
     */
    public AssignTaxDto $body;

    /**
     * @param array{
     *   body: AssignTaxDto,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->body = $values['body'];
    }
}

File: sdks/ruby/lib/novabilling/taxes/types/assign_to_plan_taxes_request.rb
# frozen_string_literal: true

module Novabilling
  module Taxes
    module Types
      class AssignToPlanTaxesRequest < Internal::Types::Model
        field :plan_id, -> { String }, optional: false, nullable: false, api_name: "planId"
        field :body, -> { Novabilling::Types::AssignTaxDto }, optional: false, nullable: false
      end
    end
  end
end
